package.json

```diff
@@ -2,23 +2,53 @@
   "name": "@superfaceai/passport-twitter-oauth2",
   "version": "1.1.0",
   "description": "Twitter OAuth 2.0 authentication strategy for Passport.",
-  "main": "./lib",
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
   "repository": "git@github.com:superfaceai/passport-twitter-oauth2.git",
   "author": "Superface Team",
   "license": "MIT",
+  "private": false,
+  "files": [
+    "dist/**/*"
+  ],
   "scripts": {
-    "test": "mocha --require ./test/bootstrap/node --recursive",
-    "format": "prettier --check ./{lib/**/*.js,test/**/*.test.js}",
-    "format:fix": "prettier --write ./{lib/**/*.js,test/**/*.test.js}"
+    "build": "tsc -p tsconfig.release.json --outDir dist",
+    "clean": "rimraf dist/",
+    "test": "ts-mocha -r ./test/bootstrap/node.js --recursive 'test/**/*.ts'",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "format": "prettier --check ./{src/**/*.ts,test/**/*.test.ts}",
+    "format:fix": "prettier --write ./{src/**/*.ts,test/**/*.test.ts}",
+    "prebuild": "yarn clean",
+    "prepack": "yarn build"
   },
   "devDependencies": {
-    "chai": "^4.3.4",
+    "@types/chai": "^4.3.3",
+    "@types/expect": "^24.3.0",
+    "@types/mocha": "^10.0.0",
+    "@types/node": "^18.11.9",
+    "@types/passport": "^1.0.11",
+    "@types/passport-oauth2": "^1.4.11",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.27.1",
+    "chai": "^4.3.6",
     "chai-passport-strategy": "^3.0.0",
-    "mocha": "^9.2.0",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-typescript": "^3.5.2",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.1.4",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "mocha": "^10.1.0",
     "prettier": "^2.5.1",
-    "proxyquire": "^2.1.3"
+    "proxyquire": "^2.1.3",
+    "ts-mocha": "^10.0.0",
+    "ts-node": "^10.9.1",
```

Comment:
suggestion: I think you can remove direct ts-node dependency since it's handled by ts-mocha.

Reply:
fixed